src/qml/views/keyframes/Parameter.qml

```diff
@@ -161,6 +161,7 @@ Item {
             onInterpolationChanged: canvas.requestPaint()
             Component.onCompleted: {
                 position = (filter.in - producer.in) + model.frame
+                updateX()
```

Comment:
It was my intention that updateX() would be a private function. Maybe it should have an underscore prefix. Does it work to move this call into Keyframe.qml onCompleted?

Reply:
Yes, just checked and it works as well. Added more commits to address this.